www/index.md

```diff
@@ -31,6 +31,21 @@ IE11 compatible
 By removing these arbitrary constraints htmx completes HTML as a 
 [hypertext](https://en.wikipedia.org/wiki/Hypertext)
 
+## HTML over the wire
+
+Serializing data to JSON on the server and parsing the JSON on the client is the way 
+many frameworks (Angular, React Vue) work today.
+
+But maybe there is a simpler way?
+
+Why not keep the client simple and stateless?
+
+htmx transfer html fragments and thus simplifies how modern web application
```

Comment:
I think it might read better as:
"With htmx, you transfer html fragments thereby simplifiying how modern web applications"